dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.optional.jaxb-2.3.feature

```diff
@@ -0,0 +1,19 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.optional.jaxb-2.3
+visibility=private
+singleton=true
+IBM-App-ForceRestart: uninstall, install
+IBM-Process-Types: client, server
+Subsystem-Name: Java XML Bindings 2.3 for Java 9 and above
+IBM-API-Package: \
```

Comment:
I think you can remove these since you are pulling them in via the com.ibm.websphere.appserver.internal.optional.jaxb-2.3 feature

Reply:
done

Reply:
this needs to stay here, because the internal.optional.jaxb-2.3 feature doesn't have the `IBM-API-Package` clause

Reply:
These lines kept there